consultation_analyser/consultations/urls.py

```diff
@@ -13,17 +13,32 @@
     path("consultations/", consultations.index, name="consultations"),
     path("consultations/new/", consultations.new, name="new_consultation"),
     path("consultations/<str:consultation_slug>/", consultations.show, name="consultation"),
+    path(
+        "consultations/<str:consultation_slug>/runs/<str:processing_run_slug>/",
+        consultations.show,
+        name="consultation_run",
+    ),
     path("schema/<str:schema_name>.json", schema.raw_schema),
     path(
         "consultations/<str:consultation_slug>/sections/<str:section_slug>/questions/<str:question_slug>/",
         questions.show,
         name="show_question",
     ),
+    path(
+        "consultations/<str:consultation_slug>/runs/<str:processing_run_slug>/sections/<str:section_slug>/questions/<str:question_slug>/",
+        questions.show,
+        name="show_question_runs",
+    ),
     path(
```

Comment:
if we didn't support displaying consultations without themes we could dispense with these! but not now